src/Microsoft.AspNet.Authentication/AuthenticationServiceCollectionExtensions.cs

```diff
@@ -12,6 +12,7 @@ public static class AuthenticationServiceCollectionExtensions
     {
         public static IServiceCollection AddAuthentication([NotNull] this IServiceCollection services)
         {
+            services.AddCaching();
```

Comment:
Okay, I'll revert this change but...

Where are we supposed to register caching services?
* In a new `services.AddOpenIdConnectAuthentication()` extension that you would have to call from your Startup.cs (would be rather weird IMO).
* Directly in Startup.cs: the developer would just have to manually call `services.AddCaching()`. :smile: